tracim/tracim/config/app_cfg.py

```diff
@@ -392,8 +392,9 @@ def __init__(self):
             'email.reply.use_txt_parsing',
             True,
         ))
-        self.EMAIL_REPLY_FILELOCK_PATH = tg.config.get(
-            'email.reply.filelock_path',
+        self.EMAIL_REPLY_LOCKFILE_PATH = tg.config.get(
+            'email.reply.lockfile_path',
+            '/tmp/tracim/email_fetcher.lock'
```

Comment:
why not use `%(here)s/tmp/email_fetcher.lock` ? `/tmp/tracim/email_fetcher.lock` may result in conflict from different instances

Reply:
%(here)s isn't interpreted correctly here. It will create a '%(here)s' folder...